tests/jballerina-unit-test/src/test/resources/test-src/query/simple-query-with-var-type.bal

```diff
@@ -217,5 +228,15 @@ function testQueryExpressionWithSpreadOperatorV2() returns Teacher[]{
                    teacherId: "TER1200"
             };
 
-    return  outputPersonList;
+    return outputPersonList;
+}
+
+public function testQueryWithStream() returns int[]|error {
+    NumberGenerator numGen = new;
+    var numberStream = new stream<int, error>(numGen);
```

Comment:
What will happen if the iterator object next method does not return an error but the stream has the errorType as the completionType?

Reply:
It passes without throwing any error